File: testprojects/master_scala_main/subgraphs/everythingSG/code/src/main/scala/orc/scala/everythingsg/Lookup_1.scala
package orc.scala.everythingsg

import io.prophecy.libs._
import config.ConfigStore._
import udfs.UDFs._
import udfs._
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._

object Lookup_1 {

  def apply(spark: SparkSession, in0: DataFrame): Unit =
    createLookup("Lookup_1",
                 in0,
                 spark,
                 List("customer_id", "first_name"),
                 "last_name",
                 "phone"
    )

}

File: testprojects/python_main/pipelines/LIVY_PYTHON/code/livy_python/graph/Lookup_1.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from prophecy.libs import typed_lit
from livy_python.config.ConfigStore import *
from livy_python.udfs.UDFs import *

def Lookup_1(spark: SparkSession, in0: DataFrame):
    keyColumns = ['''variable''']
    valueColumns = ['''value''', '''unit''']
    createLookup("Lookup1", in0, spark, keyColumns, valueColumns)
